frontend/src/store/facilityBooking/action.ts

```diff
@@ -219,7 +225,11 @@ export const setViewDates = (newDate: Date) => (dispatch: Dispatch<ActionTypes>)
   dispatch({ type: FACILITY_ACTIONS.SET_VIEW_FACILITY_END_DATE, ViewEndDate: endDate })
 }
 
-// currentMode TRUE == view bookings || FALSE == view availabilities
+/**
+ * Sets the current viewing mode in facilities
+ * @param currentMode (boolean) TRUE == view bookings, FALSE == view availabilities
```

Comment:
change the notation abit cos it might be confusing
```suggestion
 * @param currentMode (boolean) TRUE -> view bookings, FALSE -> view availabilities
```